### ISSUE TITLE
Error building `develop` via CocoaPods

### ISSUE THREAD

**Issue by akovalov** [mousebird/WhirlyGlobe#1547]
Here are 2 errors generated by Xcode when using latest version from `develop` branch installed via Cocoapods when using `use_frameworks!` attribute.
Changing to quotes like suggested "Fix-it" solves the issue. 
It does not trigger an error in AutoTester.xcodeproj and in AutoTesterPod.xcworkspace, because there is no `use_framework!` attribute.

<img width="1352" alt="Screenshot 2022-06-27 at 11 16 20" src="https://user-images.githubusercontent.com/9693643/175898749-00785fb3-0116-4b8b-90e3-ef85689d72c3.png">
<img width="1360" alt="Screenshot 2022-06-27 at 11 19 25" src="https://user-images.githubusercontent.com/9693643/175898768-53fb2e80-f709-445b-886e-37a3fcd741d9.png">

**Comment by sjg-wdw** [comment]
Roping in @TimSylvester 
I'd suggest using the main branch unless there's something you really want in develop.

**Comment by TimSylvester** [comment]
Yeah, nobody is maintaining it at that level of detail right now.

At one point it was building as a pod with and without frameworks, as well as in a submodule and as an xcframework, but keeping all of that working is hard enough without the Xcode breaking-change treadmill.

If applying the recommended fixes makes it work with and without `use_framework!` then we can merge a PR of that.  But we'll want to check the others before releasing.  If I remember correctly, quotes is the now "correct" way of including something within the project for source files and private headers, and brackets would imply that it's external.

**Comment by akovalov** [comment]
@TimSylvester I understand. I didn't submit a PR for this because thought that it would be better for you to check it and I saw changes with imports in other branch as well. So if you will decide to resolve it would be great. It's fine if not, I'll get it working with removing it manually on each pod install. Thank you.
